### PR TITLE
Add sharing is caring

### PR DESCRIPTION
**Pull request by olejnjak**

### CODE REVIEW

**Comment by janmisar** [comment]
@olejnjak Please make it conflict free 😇

**Comment by olejnjak** [comment]
@janmisar Fixed

File: README.md [hunk 1]
```diff
@@ -35,8 +35,8 @@ Simply retrieve location over time. With custom manager settings
 ReactiveLocation.singleLocationProducer { manager in
 	manager.distanceFilter = 1000
 	manager.desiredAccuracy = kCLLocationAccuracyBest
-}.startWithNext { location in
-    print(location)            
+}.startWithNext { [unowned self] location in
```
Comment (janmisar) [inline README.md]: Master branch is Swift 3 & ReactiveSwift so we should provide example in new syntax `startWithValues`.

**Review by janmisar — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (janmisar) [reply README.md]: I should notice before.. sorry 😇

Comment (olejnjak) [reply README.md]: Good point

**Comment by olejnjak** [comment]
@janmisar Done

**Pull request merged**